### DISCUSSION TITLE
Stuck on loading

### DISCUSSION THREAD

**Discussion by Calebmoyston** [vercel/community#3023] [Bug Report]
### Page to Investigate

https://vercel.com/new

### Steps to Reproduce

<img width="520" alt="Screenshot 2023-06-27 153049" src="https://github.com/vercel/community/assets/130251247/be5eb40d-4ac3-4fb0-959f-55d4ed2866f3">
I've been stuck on my account loading for the past hour. Any idea on how to fix this?

**Comment by amyegan** [comment]
Hi @Calebmoyston. Are you still having this problem?
